### DISCUSSION TITLE
Reschedule Community Office Hour to Morning Time

### DISCUSSION THREAD

**Discussion by nicoprow** [eclipse-tractusx/eclipse-tractusx.github.io#1091] [Polls]
Currently, the community office hours are taking place every friday afternoon German time. This can be inconvenient for people in other time zones, especially for some contributors that are located in Asia.

**Comment by bosserf** [comment]
I prefer to keep the 1300 h, but might be able to make 1000 h

**Comment by ma3u** [comment]
Here the link to the Office Hour:
https://eclipse-tractusx.github.io/community/open-meetings#Community%20Office%20Hour

Every Friday effective 16. Feb 2024 until 31. Dec 2024 from 01:05 pm to 02:00 pm CEST

**Comment by tom-rm-meyer-ISST** [comment]
Also keeping the slot is fine for me (in case one wants to consider that)

**Comment by danielmiehle** [comment]
@stephanbcbauer can you please prepare an update for the open meeting page and move the time slot to 10 am CEST?

**Comment by matbmoser** [comment]
Closing discussion, sorry folks in America....

**Reply by matbmoser** [reply]
Maybe we can change in the future, 13 was also not so cool anyway

**Comment by stephanbcbauer** [comment]
@danielmiehle i will merge this PR with end of the year https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/1112
